schema.graphql

```diff
@@ -0,0 +1,14 @@
+type Nft @entity(immutable: false) {
+  id: ID!
+  address: String!
+  tokenId: BigInt!
+
+  currentOwner: Account!
+  previousOwner: Account!
+  originalOwner: Account!
+}
+
+type Account @entity(immutable: false) {
+  id: ID!
+  nfts: [Nft!]! @derivedFrom(field: "originalOwner")
```

Comment:
considering the merging owner fields to only `owner`, we should update the refs here too 

```suggestion
  nfts: [Nft!]! @derivedFrom(field: "owner")
```